File: Today_Q/220425/multiORPlus.js
function solution(s){

    answer = 0

    for(let i=0;i<s.length;i++){
        item=parseInt(s[i])
        console.log("item",item)
        if(!answer||item<2){
            answer+=item
        }else{
            answer*=item
        }
    }

    console.log("ee", answer)

    return answer

}

console.log(solution("567"))

File: Today_Q/220425/multiORPlus.py
def solution(s):

    answer = 0

    for i in range(len(s)):
        x=int(s[i])
        if x<2 or not answer:
            print("+",s[i])
            answer+=x
        else:
            print("*",s[i])
            answer*=x

    print("res",answer)
    pass


if __name__ == '__main__':

    n = "567"

    palinList = solution(n)